department-of-veterans-affairs/vets-website
# Fixes defect with Material Type Filter not working

## Description
closes department-of-veterans-affairs/va.gov-cms/issues/11445
Paginator was not correctly showing pages at the end of the page list, resulting in PACT act and other documents not being visible

Several bugs around the Outreach material page paginator were noticed while working on this, but those are to be addressed in a follow up ticket

## Original issue(s)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11445


## Testing done
local visual testing
Steps to QA:
- navigate to http://localhost:3002/outreach-and-events/outreach-materials/
- set file type filter to poster
- observe that there are 4 pages, including the PACT act documents on the final page

## Screenshots
<img width="755" alt="Screen Shot 2022-11-09 at 9 27 17 AM" src="https://user-images.githubusercontent.com/61624970/200856660-f77f4633-3d16-4277-ae69-6954fb618b3f.png">
<img width="755" alt="Screen Shot 2022-11-09 at 9 27 36 AM" src="https://user-images.githubusercontent.com/61624970/200856662-d18d1fac-b5a1-46b8-9a5b-91af28a1ccbc.png">


## Acceptance criteria
- [ ] PACT act documentation is visible under the correct file type filters

## Definition of done
